backend/plugin/db/pg/sync.go

```diff
@@ -402,6 +403,8 @@ func getTableColumns(txn *sql.Tx) (map[db.TableKey][]*storepb.ColumnMetadata, er
 			column.Type = fmt.Sprintf("%s.%s", udtSchema.String, udtName.String)
 		case "ARRAY":
 			column.Type = udtName.String
+		case "character", "character varying":
```

Comment:
> [If data_type identifies a character or bit string type, the declared maximum length; null for all other data types or 
if no maximum length was declared.](https://www.postgresql.org/docs/current/infoschema-columns.html)

```bash
h3n4l@127:h3n4l> CREATE TABLE t(a BIT(3), b BIT VARYING(5));
CREATE TABLE
Time: 0.042s
h3n4l@127:h3n4l> SELECT cols.data_type, cols.character_maximum_length FROM INFORMATION_SCHEMA.COLUMNS AS cols WHERE table_name = 't';
+-------------+--------------------------+
| data_type   | character_maximum_length |
|-------------+--------------------------|
| bit         | 3                        |
| bit varying | 5                        |
+-------------+--------------------------+
SELECT 2
Time: 0.018s
h3n4l@127:h3n4l>
```

Should we consider bit string type?

Reply:
https://www.postgresql.org/docs/current/datatype-bit.html